main.py

```diff
@@ -0,0 +1,34 @@
+import random
+
+def Names():
+    """
+    The function asks the user for names, checks them, and returns them.
+    """
+    name_list = input("Please put in a list of names separated by spaces: ")
+    name_list_final = list(set(name_list.split()))
+
+    while len(name_list_final) < 2:
+        name_list = input("Please input more than one name please. ")
+        name_list_final = list(set(name_list.split()))
+
+    return name_list_final
+    
+def Number(name_list_final):
+    """ 
+    This function will ask the user the number of names they want from the list

```

Comment:
Small inconsistency with documentation: "The function..." on line 5 vs "This function..." on line 18. Pick one convention and stick to it